apis/v1alpha2/grpcroute_types.go

```diff
@@ -90,7 +90,7 @@ type GRPCRouteStatus struct {
 type GRPCRouteSpec struct {
 	CommonRouteSpec `json:",inline"`
 
-	// Hostnames defines a set of hostname to match against the GRPC
```

Comment:
Wait, why did we change this? This is godoc, the godoc for the attribute should be the same case as the code?

Reply:
Reverted the change for now, I think there may have been an upstream effort to make godocs text match the user-facing capitalization, but a quick run through k/k shows that's not very widespread right now. @thockin might have more context.